CHANGELOG.md

```diff
@@ -6,6 +6,8 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
 ## [Unreleased]
+### Added
+- Add field variations values to `productQuery`
```

Comment:
`a field values to the variations field of `productQuery`` sounds more understandable, don't you think?

Reply:
Of rock! 👍